index.js

```diff
@@ -0,0 +1,80 @@
+// function takes in an integer and outputs a random integer between 0 & input
+const randomInt = (max) => {
+  return Math.round(Math.random() * (max))
+}
+
+// function accepts a integer of the size of the gameboard's sides
+// it outputs a 2-dimensional array, initialized with all `0` values
+// `0` represents a dead cell, `1` represents an alive cell
```

Comment:
I like that you simplified it to a 0 is dead and alive is 1. I made each one an object with a property isAlive ... this seems way cleaner.

Reply:
Thanks! The idea was to use it in conditionals (ala `if (!cell)`) - but I forgot there could be *other* falsey values too - so it wasn't as helpful as I meant it to be haha but still works i suppose!